tests/end-to-end/external-solution-integration/Makefile

```diff
@@ -14,9 +18,17 @@ test:
 	@echo "Tests are not to be run at this stage."
 
 clustertest:
-	cat testdefinition.ko.yaml| KYMADOMAIN=$(shell kubectl get configmap net-global-overrides -n kyma-installer -o jsonpath='{.data.global\.ingress\.domainName}') envsubst | ko apply -n kyma-system -f -
+	rm .ko.yaml || true
+	ln -s .ko.curl.yaml .ko.yaml
+	cat testdefinition.ko.yaml | KYMADOMAIN=$(shell kubectl get configmap net-global-overrides -n kyma-installer -o jsonpath='{.data.global\.ingress\.domainName}') envsubst | ko apply -n kyma-system -f -
 	kyma test run core-test-external-solution --watch
 
+debug-local:
+	kubectl delete pod -n kyma-system -l app=core-test-external-solution
```

Comment:
If the pod does not exist for some reason, this will break the execution of the rest of the step. Let's use `--ignore-not-found`